proxy/coco/src/peer.rs

```diff
@@ -29,13 +29,15 @@ pub mod gossip;
 pub mod include;
 
 mod run_state;
-pub use run_state::{AnnounceConfig, Config as RunConfig, Event, Status, SyncConfig};
+pub use run_state::{config as run_config, Config as RunConfig, Event, Status};
 
 mod subroutines;
 use subroutines::Subroutines;
 
 mod sync;
 
+pub mod waiting_room;
```

Comment:
What's the reason to `pub` this one?

Reply:
I don't think the functions inside need to be private. I find that it's easier to work with libraries when things global things are made public, like modules, and then if functions and fields need to hidden make those private.

Would you prefer it was private?

Reply:
It is preferrably to keep the exposed surface smaller as it reduces the coginitive overhead for consumers. In this case esp. the waiting room handling is mostly an internal concern to the peer and should not be general utility.